src/app/(proper_react)/redesign/(authenticated)/user/welcome/View.tsx

```diff
@@ -26,16 +27,21 @@ export type Props = {
   dataBrokerCount: number;
   breachesTotalCount: number;
   stepId?: StepId;
+  skipInitialStep?: boolean;
 };
 
 export const View = ({
   user,
   dataBrokerCount,
   breachesTotalCount,
   stepId = "getStarted",
+  skipInitialStep,
```

Comment:
Nit, but it might be good if the caller doesn't need to remember what the initial step is (or maybe even why you'd want to skip it). So a more descriptive name, like `skipGetStarted` or `startAtEnterInfo` or something like that could be something to consider?

Reply:
Actually, can't we just derive this from `stepId` being set to `"enterInfo"` in the first place? With the added benefit that that would no longer be an API in runtime code that's only used in tests :)

Reply:
I’ve added the additional prop to distinguish between setting the initial step of the “long” onboarding vs the shortened version. But since that’s just helpful for writing tests some minor changes there do the same thing. Updated in [efbeb7a](https://github.com/mozilla/blurts-server/pull/3541/commits/efbeb7a01c3378bf9831660b810a5218364665cb).